src/main/docker/Dockerfile

```diff
@@ -37,6 +37,7 @@ ENV CACHE_DIR="-Dvertx.cacheDirBase=/tmp"
 ENV CONFIG_FILE="-Dvertx-config-path=/etc/${project.artifactId}/${project.artifactId}.properties"
 ENV JAVA_TOOL_OPTIONS="${jdwp.client.config}"
 ENV JAR_PATH="/usr/local/${project.artifactId}/${project.artifactId}-${project.version}.jar"
-CMD ["sh", "-c", "java $CACHE_DIR $CONFIG_FILE -Xmx2g -jar $JAR_PATH"]
+ENV LOGBACK_PATH="-Dlogback.configurationFile=/etc/fester/logback.xml"
```

Comment:
/etc/fester/logback.xml does not exist. It was my intention to set a default ENV value to that path, but since it doesn't exist, it just defaults to load the logback.xml file within the jar.

The ENV allows me to override externally with an HTTP URL, so in my k8s deployment, I would set up the following ENV:

          - name: LOGBACK_PATH
            value: "-Dlogback.configurationFile=https://raw.githubusercontent.com/UCLALibrary/fester/IIIF-834/src/main/resources/logback-fluency.xml"

This will make fester load the logback-fluency.xml file associated with this PR if the link returns a 200.

Reply:
Why not just not set the ENV in the Dockerfile but include the var in the CMD that starts the Jar? If LOGBACK_PATH is not set it should just be empty in the CMD, right? If you've supplied it via external ENV then it should have your value.

Reply:
Alternatively, you could just set `ENV LOGBACK_PATH=""` in the Dockerfile, but I think it should work without setting it at all.